README.arm.md

```diff
@@ -23,10 +33,7 @@ If you get SIGILL during sysimg.o creation, it is likely that your cpu
 does not support VFP.  File an issue on the Julia issue tracker with
```

Comment:
I don't think this is an issue anymore, especially not for native build. Mentioning that `/proc/cpuinfo` content should be included for ARM bug report would be good though.